### ISSUE TITLE
About page splash image lacks alt text

### ISSUE THREAD

**Issue by noraegloff** [LafayetteCollegeLibraries/spot#852]
[About page](https://ldr.lafayette.edu/about) splash image lacks alt text. 
<img> elements must either have alternate text or a role of none or presentation.

Associated elements/source code viewed in test: 
`<img class="splash-image img-responsive" src="/assets/about-splash-4d52fb3a7f43f054c36505198844583b9fef517fa9dcf179ecd4422b1ead1960.jpg">`

**Comment by noraegloff** [comment]
Deque aXe tools severity level = **critical**
